operations/mimir/rollout-operator.libsonnet

```diff
@@ -108,6 +125,13 @@
           policyRule.withVerbs(['get', 'patch']),
         ] else []
       )
+      + (
+        if zpdb_enabled then [
```

Comment:
I think we should always enable these rules provided the rollout-operator supports ZPDBs - otherwise, won't the rollout-operator fail to start because it won't have permission to watch ZPDBs?